llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -1731,6 +1735,16 @@ X86TargetLowering::X86TargetLowering(const X86TargetMachine &TM,
     for (auto VT : { MVT::v1i1, MVT::v2i1, MVT::v4i1, MVT::v8i1 })
       setOperationAction(ISD::EXTRACT_SUBVECTOR, VT, Custom);
   }
+  if (Subtarget.hasDQI() && Subtarget.hasVLX()) {
```

Comment:
I'm not qualified to review this line, as I don't know what `DQI` or `VLX` are.

Reply:
It's mapped to `avx512dq` and `avx512vl` features.